checkov/common/output/secrets_record.py

```diff
@@ -57,7 +57,10 @@ def __init__(self,
                  definition_context_file_path: Optional[str] = None,
                  validation_status: Optional[str] = None,
                  added_commit_hash: Optional[str] = None,
-                 removed_commit_hash: Optional[str] = None
+                 removed_commit_hash: Optional[str] = None,
+                 created_by: Optional[str] = None,
+                 removed_date: Optional[str] = None,
+                 create_date: Optional[str] = None
```

Comment:
this more about consistency in wording, but `added_by` and `added_date` would fit better to  `added_commit_hash`
This would mean to change in all the places you touched.

Reply:
this would mean to change it also in platform + in the db migration
I can fix it but this will be the overhead 😅